<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;./images/loginBack.jpg&quot;)</string>
  </property>
  <widget class="QPushButton" name="log_login_butt">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(21, 161, 116)
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="log_pass_line">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>225</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="login_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>99</y>
     <width>400</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10%;
font: &quot;Ubuntu Mono&quot;;
background-color: rgb(150, 150, 150);
border-style: solid;
border-color: rgb(32, 74, 135);
font-family: &quot;Ubuntu Condensed&quot;;
font-size: 75</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="email_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>101</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border: rgb(255, 255, 255)
</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="pass_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>225</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="log_email_line">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="log_reg_butt">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 76, 84);
color: rgb(0, 0, 0)
</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>160</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>email_label</zorder>
  <zorder>log_login_butt</zorder>
  <zorder>log_pass_line</zorder>
  <zorder>login_label</zorder>
  <zorder>pass_label</zorder>
  <zorder>log_email_line</zorder>
  <zorder>log_reg_butt</zorder>
  <zorder>warning</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
